text_recognizer/data/iam.py

```diff
@@ -2,23 +2,27 @@
 import argparse
 import os
 from pathlib import Path
-from typing import Dict, List
+from typing import Dict, List, Optional
+import xml
 import zipfile
 
 from boltons.cacheutils import cachedproperty
 from defusedxml import ElementTree
 import toml
+from PIL import Image, ImageOps
 
 from text_recognizer.data.base_data_module import _download_raw_dataset, BaseDataModule, load_and_print_info
+from text_recognizer import util
 import text_recognizer.metadata.iam as metadata
+from text_recognizer.metadata.iam_paragraphs import NEW_LINE_TOKEN
 
 
 METADATA_FILENAME = metadata.METADATA_FILENAME
 DL_DATA_DIRNAME = metadata.DL_DATA_DIRNAME
 EXTRACTED_DATASET_DIRNAME = metadata.EXTRACTED_DATASET_DIRNAME
 
 
-class IAM(BaseDataModule):
+class IAM:
```

Comment:
we don't implement the data split dataloaders, so there is no need for IAM to be derived from `BaseDataModule`